changelogs/client_server/newsfragments/1357.clarification

```diff
@@ -0,0 +1 @@
+Add link to the unstable spec to the README.
```

Comment:
This will need to go under the `changelogs/internal/newsfragments` directory instead.

(Let me know if the contributing documentation wasn't clear or could be improved!)